### PR TITLE
{lang}[GCC/9.3.0] LuaJIT2-OpenResty v2.1-20220411

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
This seems very confusing, especially since the official LuaJIT also has a version 2.x. I'm not really sure how to get around this, maybe we should name this one something like `LuaJIT-OpenResty`? Let me also ask the other maintainers.

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This is the LuaJIT fork from openresty, not to be mixed up
+# with the original LuaJIT version!
+# Autor: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = "LuaJIT2"
```
Comment (bedroge) [inline easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:7]: Okay, let's indeed go for `LuaJIT-OpenResty` to make a clear distinction with the existing `LuaJIT`:

```suggestion
name = "LuaJIT-OpenResty"
```

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+version = "v2.1-20220411"
```
Comment (bedroge) [inline easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:8]:
```suggestion
version = "2.1-20220411"
```

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+description = """openresty/luajit2 - OpenResty's maintained branch of LuaJIT.
+LuaJIT is a Just-In-Time Compiler (JIT) for the Lua
+programming language. Lua is a powerful, dynamic and light-weight programming
+language. It may be embedded or used as a general-purpose, stand-alone
+language. """
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/openresty/luajit2/archive']
+sources = ['%(version)s.tar.gz']
```
Comment (bedroge) [inline easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:20]:
```suggestion
sources = ['v%(version)s.tar.gz']
```

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+checksums = ['d3f2c870f8f88477b01726b32accab30f6e5d57ae59c5ec87374ff73d0794316']
+
+skipsteps = ['configure']
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ["bin/luajit"],
+    'dirs': []
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:30]: Let's also add a sanity check command:

```suggestion

sanity_check_commands = ['luajit -v']

```

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:7]: That makes sense. I was not quite sure what was the best way forward here. Thanks!

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb (hunk 2 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:8]: Changed, thanks for spotting this.

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb (hunk 3 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:20]: As above

File: easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb (hunk 4 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb:30]: I must have overlooked that. Thanks.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.4
See https://gist.github.com/9623148f73ba5108889604d3cd1d2238 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Review by bedroge — APPROVED** [review]

**Comment by bedroge** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**